spec/amber/cli/build_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "../../spec_helper"
```

Comment:
The check for the `run_build_tests` flag is gone - so this is running by default. Is that intended?

Reply:
I am planning to add the flag when I am done I had to keep reminding myself to run the spec with the flag :)

Reply:
🎏  Are up!